application/core/utils/fileExtensions.php

```diff
@@ -1,17 +1,23 @@
 <?php
+
 /**
  * File extensions by MIME types.
  *
  * Include Yii and extend with new one
  * @version 1.0
  */
+
 return array_merge(
-	require(Yii::getPathOfAlias('system.utils.fileExtensions').'.php'), // Yii framework
-	/* iphone image : issue #15624 */
-	array(
-		'image/heif'=>'heif',
-		'image/heif-sequence'=>'heifs',
-		'image/heic'=>'heic',
-		'image/heic-sequence'=>'heics',
-	)
+    require(Yii::getPathOfAlias('system.utils.fileExtensions') . '.php'), // Yii framework
+    array(
+        /* iphone image : issue #15624 */
+        'image/heif' => 'heif',
+        'image/heif-sequence' => 'heifs',
+        'image/heic' => 'heic',
+        'image/heic-sequence' => 'heics',
+        /* xml as text/xml : #18353 */
+        'text/xml' => 'xml',
+        /* svg as image/xml : #18347 */
+        'image/svg' => 'svg',
```

Comment:
Hi Denis,

this task/bug was about text/xml. Is it really needed to also 'image/svg' => 'svg' for that bugfix?
Could you reproduce the bug and did you tested it by yourself after fixing it?
My problem is that i do not have ngingx set up right now for first reprocuding the bug and afterwards test if it still exist.

Greetings Patricia

Reply:
2 issue in same file :  Fixed issue #[18347](https://bugs.limesurvey.org/view.php?id=18347): Unable to check extension of this file type image/svg 


Yes : on my debian 11, the file sent by user show as `image/svg`, i'm on nginx too.

I have exactly the same issue than reporter.
![Capture d’écran du 2022-09-30 12-14-19](https://user-images.githubusercontent.com/1439428/193248543-f6761360-df56-4ab9-b7e1-671f1a1062a3.png)

Reply:
PS : it's notb related to nginx, you have the same issue with apache. It's related to mime_type / fileinfo.

If fileinfo/mimetype return image/svg : we don't set it as svg.

Reply:
Thanks Deni. So testing is ok from my side. Will merge it :)

Reply:
You use apache on ? for testing ?